doc/source/release-notes/1.17.2.rst

```diff
@@ -0,0 +1,67 @@
+.. _1.17.2:
+
+================================================================================
+Percona Monitoring and Management 1.17.2
+================================================================================
+
+:Date: October 23, 2019
+
+*PMM* (`Percona Monitoring and Management <https://www.percona.com/doc/percona-monitoring-and-management/index.html>`_) is a free and open-source platform for managing and monitoring |MySQL|, |MongoDB|, and |postgresql| performance. You can run *PMM* in your own environment for maximum security and reliability. It provides thorough time-based analysis for |MySQL|, |MongoDB|, and |postgresql| servers to ensure that your data works as efficiently as possible.
+
+One of the improvements in this release is a substantial size increase for the AMI images, which allows now to reserve space for about 10 instances with 30 days retention. Another improvement is the updated Tooltips for the MySQL InnoDB Compression dashboard by providing a description of what the graphs display, along with links to related documentation resources:
```

Comment:
Note: this is for NEW installations only.  need to mentioned + link to documentation  how to increase if needed